test/servico/Cst22Test.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package servico;

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['package' should be separated from previous statement.](https://www.codacy.com/app/calebepb/substituicao/file/2770060921/issues/source?bid=3308597&fileBranchId=3351578#l6)